.babelrc

```diff
@@ -1,50 +1,88 @@
 {
   "env": {
-    "test": {
+    "development": {
       "presets": [
-        ["env", {"targets": { "node": "current" }}],
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
-      ],
-      "plugins": [
-        "babel-plugin-rewire",
-        ["module-resolver", {
-          "alias": {
-            "preact-compat": "react"
+      ]
+    },
+    "production": {
+      "presets": [
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
           }
-        }]
+        ],
+        "react",
+        "stage-2"
       ]
     },
-    "development": {
+    "umd": {
       "presets": [
-        ["env", { "targets": { "browsers": ["last 2 versions", "ie >= 10"] } }],
+        [
+          "env",
+          {
+            "modules": false,
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
       ]
     },
-    "production": {
+    "es": {
       "presets": [
-        ["env", { "targets": { "browsers": ["last 2 versions", "ie >= 10"] } }],
+        [
+          "env",
+          {
+            "modules": false,
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
       ],
       "plugins": [
-        "transform-react-remove-prop-types",
-        "transform-react-constant-elements",
-        "transform-react-pure-class-to-function"
+        [
+          "inline-replace-variables",
```

Comment:
It's only useful for the `es` env?

Reply:
Yes (`es` env is used for `cjs` and `es`).

In UMD, we replace `__DEV__` depending on the bundle type with Rollup. In CJS and ES, we replace `__DEV__` depending on the Node environment.